test/makeLatexReportRegression.m

```diff
@@ -0,0 +1,231 @@
+function makeLatexReportRegression(currentStatus, output, otherStatus)
+% generate a LaTeX report
+%
+% 
+    if ~exist('output','var')
```

Comment:
I think it's a bit weird to have `output` optional, while `otherStatus` is a required option. I would make `output` the last option to provide such that a defaulted value makes sense.